tailwind.config.js

```diff
@@ -28,19 +28,14 @@ module.exports = {
   theme: {
     extend: {
       screens: {
-        'xl-': { max: '1535px' },
-        'lg-': { max: '1279px' },
-        'md-': { max: '1023px' },
-        'sm-': { max: '767px' },
         'sm+': { min: '640px' },
```

Comment:
I've never used this `-` / `+` for responsive Tailwind before and I must say I am a fan, means we're not forced to class them mobile first.